signature/signature.go

```diff
@@ -112,8 +112,9 @@ var _ json.Unmarshaler = (*untrustedSignature)(nil)
 func (s *untrustedSignature) UnmarshalJSON(data []byte) error {
 	err := s.strictUnmarshalJSON(data)
 	if err != nil {
-		if _, ok := err.(jsonFormatError); ok {
+		if serror, ok := err.(jsonFormatError); ok {
```

Comment:
I don't see golangci complaining about it. Why is it needed?

Reply:
S1020: when ok is true, err can't be nil

Reply:
Sure, but why are we doing this change? Instead of ignoring the error, we're now assigning it to nowhere.